spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -452,6 +456,22 @@ private void declareQueue(String beanName, Queue queue) {
 		addToAutoDeclareContext(beanName, queue);
 	}
 
+	private Exchange buildExchange(RabbitCommonProperties properties, String exchangeName) {
+		try {
+			// TODO Make the ctor public in Spring-AMQP - AMQP-695
+			Constructor<ExchangeBuilder> ctor = ExchangeBuilder.class.getDeclaredConstructor(String.class, String.class);
```

Comment:
I don't know - @sabbyanandan ?

OTOH, I don't see that it's a big deal for us to fix up later.